xml/schema/Contribute/Contribution.xml

```diff
@@ -203,6 +203,7 @@
     <import>true</import>
```

Comment:
So we kinda edit trxn_id but we are trying to move away from it as it is a payment field not a contribution field. Since this metadata is for a new UI I think it's fine to treat as readonly